index.html

```diff
@@ -30,9 +28,18 @@
 
       <div class="FadeImage"><img src="./view-of-drosia.jpg" alt="View of Drosia" class="Center"/></div>
 
+
+      
+      <center><iframe width="560" height="315" src="https://www.youtube.com/embed/IZNSk3OnKaI?si=JOLS9Rgq8LYF_vKf" 
```

Comment:
I've never seen the center tag before haha, learn something new everyday!

Turns out the tag is depreciated in html 5 so it's pretttty old.
<img width="537" alt="image" src="https://github.com/NikoTheGreeko/VillageWebsiteDrosia/assets/33900517/69c7b3d3-72c8-43f9-b68c-685579cec4fd">


Instead, on the iframe tag you can make a class and use `margin: auto` to center it horizontally.